### PR TITLE
Update: README.md

### PR DESCRIPTION
**Pull request by felixodette**
Use ruby gems as software packages system.
Install Ruby on Rails framework.
Understand Rails RESTful design and router.
Use controllers to handle requests and render empty views.
Use params from browser request in a safe way.
Use preprocessed html file with embedded Ruby code.
Use layouts and templates for shared content.
Use database migration files to maintain database schema.
Use validations for models.
Secure app from n+1 problems.
Understand what ORM is.
Write SQL queries with ActiveRecord.
Set up associations between models.
Build a webapp that requires the user to log in.
Use devise gem for authentication.
Limit access to webapp resources based on authorization rules.
Analyze in writing why you have made a coding choice using one structure over another.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -43,52 +43,54 @@
...
-
-# The settings below are suggested to provide a good initial experience
-# with RSpec, but feel free to customize to your heart's content.
-begin
-  # This allows you to limit a spec run to individual examples or groups
-  # you care about by tagging them with `:focus` metadata. When nothing
-  # is tagged with `:focus`, all examples get run. RSpec also provides
-  # aliases for `it`, `describe`, and `context` that include `:focus`
-  # metadata: `fit`, `fdescribe` and `fcontext`, respectively.
-  config.filter_run_when_matching :focus
```
Comment (Davidosky007) [inline spec/spec_helper.rb:56]: * I get this error when i try to run your tests with this command `rspec` kindly fix it 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @felixodette 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* Nice looking UI design 👍 
* Good looking readme file 👍 
* Well written PR description 👍 

## Required Changes ♻️
* Kindly record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video should include:

> a description of the project.
> a demo of the project features.
> you should also highlight some interesting piece of code or something you built that you are very proud of.
* Kindly stick to the column names visible in the ERD diagram as stated in the project requirement ( ie author_id instead of user_id)
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/spec_helper.rb (hunk 1 above)
Comment (felixodette) [reply spec/spec_helper.rb:56]: Screenshot 2022-09-23 at 14 29 20
Hey @Davidosky007 the tests run on my end. Strange!

**Review by omarsalem7 — APPROVED** [review]
Hi @felixodette 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Good UI design
✅ Well written pull request description. 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**